tests/python/gaia-ui-tests/gaiatest/apps/system/regions/cards_view.py

```diff
@@ -125,6 +97,46 @@ class Card(PageRegion):
     _close_button_locator = (By.CLASS_NAME, 'close-button')
     _screenshot_view_locator = (By.CLASS_NAME, 'screenshotView')
     _app_icon_locator = (By.CLASS_NAME, 'appIcon')
+    _title_locator = (By.CLASS_NAME, 'title')
+    _subtitle_locator = (By.CLASS_NAME, 'subtitle')
+    _close_buttons_locator = (By.CSS_SELECTOR, '.close-button')
+
+    @property
+    def title(self):
+        return self.root_element.find_element(*self._title_locator).text
+
+    @property
+    def subtitle(self):
+        return self.root_element.find_element(*self._subtitle_locator).text
+
+    @property
+    def is_displayed(self):
+        return self.root_element.is_displayed()
+
+    @property
+    def manifest_url(self):
+        return self.root_element.get_attribute('data-origin')
+
+    @property
+    def is_centered(self):
+        screen_width = int(self.marionette.execute_script('return window.innerWidth'))
+        left = self.root_element.rect['x']
+        width = self.root_element.rect['width']
+        # center of card should be within 1px of viewport center
+        return 1 >= abs(screen_width / 2 - (left + width / 2))
+
+    def wait_for_centered(self):
+        Wait(self.marionette).until(lambda m: self.is_centered)
+
+        # TODO: Remove sleep when we find a better wait
+        time.sleep(0.2)
```

Comment:
Is this wait still necessary? I remember we could remove it once we changed `is_centered`